app/assets/stylesheets/_mini_groove_song_card.css.sass

```diff
@@ -0,0 +1,25 @@
+.songcard
```

Comment:
I think that not all of you are using the same tab/space indentation.  I'd recommend you set sublime to use spaces instead of tabs and set a tab to equal 2 spaces.  It doesn't really matter as long as its the same for everyone.